### DISCUSSION TITLE
【求助】我老婆丢了，帮我找找吧

### DISCUSSION THREAD

**Discussion by jiishdhhwhahysyd** [share121/inter-knot#139] [其他]
以下是我老婆的照片[doge]
![IMG_20240809_191514](https://github.com/user-attachments/assets/ab840791-c1a7-4747-b308-931282fa8a2b)

**Comment by zhuang1142** [comment]
这个委托我接了

**Comment by AL-Pinecore** [comment]
这个委托我接了

**Comment by minbvv** [comment]
不好意思啊哥，嫂子在我这儿恐怕回不来了

**Reply by jiishdhhwhahysyd** [reply]
草！

**Comment by jiishdhhwhahysyd** [comment]
草！

**Comment by Vrey2077** [comment]
在我这呢，好得很

**Comment by capacitor1** [comment]
这个委托我接了！

，
，
委托已经完成了！你的老婆找到了，在：
[Storage:Pixiv@114065737](https://www.pixiv.net/artworks/114065737)
[File:Yandere@1138183 - 6.47MB](https://files.yande.re/image/cbf9736a1981f0a72dc18de75ba21cac/yande.re%201138183%20furina%20garter%20genshin_impact%20heterochromia%20shinyo_yukino%20sweater%20wings.jpg)

**Reply by AL-Pinecore** [reply]
666

**Reply by cooperatior** [reply]
口瓜，是顶级绳匠

**Comment by 353563** [comment]
这个委托我接了

**Comment by jiishdhhwhahysyd** [comment]
捞一下

**Comment by warning-Kirov-reporting** [comment]
诶，这不是我走丢多年的脑婆吗

**Comment by Fairy561** [comment]
这个委托我接了

**Comment by SakuraLainMaid** [comment]
这个委托我接了

**Comment by Stewiegriffin6** [comment]
这个委托我接了

**Comment by Qijoten** [comment]
这个委托我接了！

**Comment by ybj67** [comment]
楼主还在太一之梦吗？(′へ`、 )

**Comment by lizard315315** [comment]
在这呢

**Comment by niegong** [comment]
这个委托…唉等等 这不是我老婆吗😘

**Comment by wu364** [comment]
她是我老婆，不是你老婆😡

**Comment by dengyunzhe** [comment]
这个老婆我接了

**Comment by xichulangren** [comment]
这个委托我接了！

**Comment by FurinaShark** [comment]
好可爱的芙芙，收了

**Comment by YoKiiiiiiiiiiiii** [comment]
![IMG_4138](https://github.com/user-attachments/assets/0b956d0a-81d0-44f8-8db1-b330b3208543)
你老婆是这个嘛，刚从我家床上睡起来

**Comment by xtd114514** [comment]
这个委托我接了！

**Comment by Philia093-T** [comment]
什么你老婆，是我老婆！😡

**Comment by hanshui2077** [comment]
这个委托我劫了
